foundation_pose_node:
  ros__parameters:
    device: "cuda:0"
    obj_file: "/home/ccs/my-local-models/YCB-V/obj_000005/obj_000005.obj"

    # image_topic: "/camera/camera/aligned_depth_to_color/image_raw"
    # depth_topic: "/camera/camera/color/image_raw"

    # sim
    image_topic: "/zedm/image"
    depth_topic: "/zedm/depth_image"
    camera_info_topic: "/zedm/camera_info"